liquibase-standard/src/main/java/liquibase/util/ShowSummaryUtil.java

```diff
@@ -280,4 +278,29 @@ private static Writer createOutputWriter(OutputStream outputStream) throws IOExc
         String charsetName = GlobalConfiguration.OUTPUT_FILE_ENCODING.getCurrentValue();
         return new OutputStreamWriter(outputStream, charsetName);
     }
+
+    private static void writeMessage(String message, UpdateSummaryOutputEnum showSummaryOutput, OutputStream outputStream) throws LiquibaseException {
+        if (showSummaryOutput != UpdateSummaryOutputEnum.CONSOLE) {
```

Comment:
Maybe something minor, but I would prefer seeing something like this:
```
switch(showSummaryOutput) {
  case UpdateSummaryOutputEnum.CONSOLE:
    writeToOutput(outputStream, message);
    break;
  case UpdateSummaryOutputEnum.LOG:
   writeToLog(message);
    break;
  default:
    writeToLog(message);
    writeToOutput(outputStream, message);
}
```

Reply:
Done!